x/harvest/keeper/params.go

```diff
@@ -37,3 +39,14 @@ func (k Keeper) GetDelegatorSchedule(ctx sdk.Context, denom string) (types.Deleg
 	}
 	return types.DelegatorDistributionSchedule{}, false
 }
+
+// GetMoneyMarket returns the corresponding Money Market param for a specific denom
+func (k Keeper) GetMoneyMarket(ctx sdk.Context, denom string) (types.MoneyMarket, bool) {
```

Comment:
created #706  - would like to avoid calling `GetParams` when handling a `MsgBorrow`